### PR TITLE
feat(k8s): add comparision table

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Comment by tgenaitay** [comment]
LGTM! Well done. Let's get this out and iterate further according to feedback we'll get.

**Review by tgenaitay — APPROVED** [review]

File: containers/kubernetes/reference-content/understanding-differences-kapsule-kosmos.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - Available in multiple Scaleway regions (FR, AMS, WAW), allowing users to deploy applications closer to their target audience for enhanced performance.
+  - Kosmos allows users to deploy, manage, and scale containerized applications using Kubernetes without having to manage the underlying infrastructure.
+  - Automatic scaling, rolling updates, and seamless integration with other Scaleway services like Load Balancers and Object Storage.
+  - Users can manage their Kubernetes clusters through the Kubernetes API or the intuitive Scaleway console.
+
+**Kapsule is ideal for:** Developers and organizations seeking to deploy containerized applications with Kubernetes without the operational overhead of managing Kubernetes infrastructure.
+
+### Kubernetes Kosmos
+
+Kosmos is Scaleway's **multi-cloud Kubernetes solution:**, designed to operate across multiple cloud environments, empowering users to deploy applications on different cloud providers, thus offering flexibility and avoiding vendor lock-in.
```
Comment (bene2k1) [inline containers/kubernetes/reference-content/understanding-differences-kapsule-kosmos.mdx:32]:
```suggestion
Kosmos is Scaleway's **multi-cloud Kubernetes solution**, designed to operate across multiple cloud environments, empowering users to deploy applications on different cloud providers, thus offering flexibility and avoiding vendor lock-in.
```

File: containers/kubernetes/reference-content/understanding-differences-kapsule-kosmos.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+**Kapsule is ideal for:** Developers and organizations seeking to deploy containerized applications with Kubernetes without the operational overhead of managing Kubernetes infrastructure.
+
+### Kubernetes Kosmos
+
+Kosmos is Scaleway's **multi-cloud Kubernetes solution**, designed to operate across multiple cloud environments, empowering users to deploy applications on different cloud providers, thus offering flexibility and avoiding vendor lock-in.
+
+  - Available in multiple Scaleway regions (FR, AMS, WAW), allowing users to deploy applications closer to their target audience for enhanced performance.
+  - Offers dedicated control plane options, ensuring dedicated resources for managing Kubernetes clusters.
+  - Supports various node pool types, including Scaleway instances (x86 and ARM architecture) and external providers, providing users with options to tailor node configurations according to their application needs.
+  - Utilizes the Kilo Container Network Interface (CNI) plugin for efficient networking within Kubernetes clusters.
```
Comment (bene2k1) [inline containers/kubernetes/reference-content/understanding-differences-kapsule-kosmos.mdx:37]:
```suggestion
  - Uses the Kilo Container Network Interface (CNI) plugin for efficient networking within Kubernetes clusters.
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: containers/kubernetes/reference-content/understanding-differences-kapsule-kosmos.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+|:-----------------------------------:|:----------------------------------------------------------:|:-----------------------------------------------------------------------------------------------------------:|
+| Cloud                               | Scaleway only                                              | Multi-Cloud                                                                                                 |
+| Regions                             | FR, AMS, WAW                                               | FR, AMS, WAW                                                                                                |
+| Node pools                          | Scaleway Instances only, x86 or ARM                        | Scaleway Instances, x86 or ARM External pools: Scaleway elastic metal, Scaleway RiscV, External providers.. |
+| Node pools locations                | Mono AZ or Multi AZ (same region)                          | Mono-AZ, Multi-AZ, Multi-Region, Multi-Cloud                                                                |
```
Comment (bene2k1) [inline containers/kubernetes/reference-content/understanding-differences-kapsule-kosmos.mdx:48]:
```suggestion
| Node pools locations                | Mono-AZ or Multi-AZ (same region)                          | Mono-AZ, Multi-AZ, Multi-Region, Multi-Cloud                                                                |
```

File: containers/kubernetes/reference-content/understanding-differences-kapsule-kosmos.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+| Auto healing                        | ✔️                                                          | Scaleway Instances only                                                                                     |
+| Auto scaling                        | ✔️                                                          | Scaleway Instances only                                                                                     |
+| Container Storage Interface         | ✔️ Persistent volumes (Block storage) on Scaleway instances | Scaleway Instances only                                                                                     |
+| Dedicated Control Plane options     | ✔️                                                          | ✔️                                                                                                           |
+| Scaleway VPC                        | ✔️ Controlled Isolation or Full Isolation                   | ✘ No integration                                                                                            |
```
Comment (bene2k1) [inline containers/kubernetes/reference-content/understanding-differences-kapsule-kosmos.mdx:54]:
```suggestion
| Scaleway VPC                        | ✔️ Controlled isolation or full isolation                   | ✘ No integration                                                                                            |
```

**Pull request merged**